requirements_common.txt

```diff
@@ -1,3 +1,5 @@
+# workarounds for pip 20.0.2's buggy dependency resolver, which doesn't account for cross-package constraints
+pyparsing<3,>=2.0.2
```

Comment:
```suggestion
# workarounds for pip 20.0.2's buggy dependency resolver
# which doesn't account for cross-package constraints: https://pip.pypa.io/en/stable/user_guide/#changes-to-the-pip-dependency-resolver-in-20-3-2020
# matplotlib pulls in the latest pyparsing, nibabel pulls in the latest packaging, and the two latests are in conflict.
# > ERROR: packaging 21.2 has requirement pyparsing<3,>=2.0.2, but you'll have pyparsing 3.0.5 which is incompatible.
# This forces the older pip, if the user has the older pip, to behave itself.
# This is to specially support users on Ubuntu 20.04 LTS; when Ubuntu 22.04 LTS comes out, this can be removed.
pyparsing<3,>=2.0.2
```

Reply:
How about this?